gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/sampling/statistics/SamplingEvaluationConstants.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.sampling.statistics;
+
+/**
+ * Collected constants for sampling evaluation
+ */
+public class SamplingEvaluationConstants {
+
+  /**
+   * Key for density property generated by {@link GraphDensity}
+   */
+  public static final String PROPERTY_KEY_DENSITY = "density";
```

Comment:
We should compute graph density only for one graph at a time. This way every graph get the same property "density" to its GraphHead. We dont need to divide them by property key.